From 497e4c9a0cc475175cc780e5100600b3083676f8 Mon Sep 17 00:00:00 2001
From: chenglulu <chenglulu@loongson.cn>
Date: Wed, 4 Aug 2021 14:29:29 +0800
Subject: [PATCH 07/10] LoongArch: Modify the value of ISA in
 loongarch_cpu_info_table.

	gcc/config/loongarch
	* loongarch-cpus.def: Modify the value of ISA in loongarch_cpu_info_table.
---
 gcc/config/loongarch/loongarch-cpus.def | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/gcc/config/loongarch/loongarch-cpus.def b/gcc/config/loongarch/loongarch-cpus.def
index 8469b06f75c..9bd923fba97 100644
--- a/gcc/config/loongarch/loongarch-cpus.def
+++ b/gcc/config/loongarch/loongarch-cpus.def
@@ -31,7 +31,7 @@ along with GCC; see the file COPYING3.  If not see
    where the arguments are the fields of struct loongarch_cpu_info.  */
 
 /* Entries for generic ISAs.  */
-LARCH_CPU ("loongarch", PROCESSOR_LOONGARCH64, 65, 0)
-LARCH_CPU ("loongarch64", PROCESSOR_LOONGARCH64, 65, 0)
-LARCH_CPU ("gs464v", PROCESSOR_GS464V, 65, 0)
+LARCH_CPU ("loongarch", PROCESSOR_LOONGARCH64, 0, 0)
+LARCH_CPU ("loongarch64", PROCESSOR_LOONGARCH64, 0, 0)
+LARCH_CPU ("gs464v", PROCESSOR_GS464V, 0, 0)
 
-- 
2.30.1

